pkg/loader/executor.go

```diff
@@ -164,24 +159,46 @@ func (e *executor) begin() (*tx, error) {
 	return tx, nil
 }
 
+// return a wrap of sql.Tx
+func (e *executor) externPoint(t *Tx, dmls []*DML) (*Tx, []*DML) {
```

Comment:
how about adding a judgment for whether support Plugin?

Reply:
Thanks. I had fixed it.